src/target/source/codegen_c_host.cc

```diff
@@ -54,6 +54,7 @@ void CodeGenCHost::Init(bool output_ssa, bool emit_asserts, bool emit_fwd_func_d
   decl_stream << "#include \"tvm/runtime/c_runtime_api.h\"\n";
   decl_stream << "#include \"tvm/runtime/c_backend_api.h\"\n";
   decl_stream << "#include <math.h>\n";
+  decl_stream << "#include <stdbool.h>\n";
```

Comment:
why this change?

Reply:
some of the generated projects have a variable of type `bool`, and result in build error if the `stdbool.h` header is not included.